app/models/proxy.rb

```diff
@@ -269,7 +269,8 @@ def self.credentials_collection
   end
 
   def set_correct_endpoints?
-    apicast_configuration_driven_changed? || new_record?
+    # Verified
```

Comment:
is this telling something or just personal note that it was tested?

Reply:
Personal note, I thought I removed all of them 😓